public/app/features/dashboard/components/PanelEditor/FieldConfigEditor.tsx

```diff
@@ -13,6 +13,7 @@ import { getDataLinksVariableSuggestions } from '../../../panel/panellinks/link_
 import { OverrideEditor } from './OverrideEditor';
 import groupBy from 'lodash/groupBy';
 import { OptionsGroup } from './OptionsGroup';
+import { Counter } from '@grafana/ui/src/components/Tabs/Counter';
```

Comment:
this looks funky :) should we reuse this component or create a new one for this use case?

Reply:
yes, missed that, great catch :)